src/containers/editorialWorkflow/EntryPageHOC.js

```diff
@@ -15,8 +15,9 @@ export default function EntryPageHOC(EntryPage) {
   function mapStateToProps(state, ownProps) {
     const { collections } = state;
     const isEditorialWorkflow = (state.config.get('publish_mode') === EDITORIAL_WORKFLOW);
-    const returnObj = { isEditorialWorkflow };
+    const returnObj = { isEditorialWorkflow, showDelete: true };
```

Comment:
Currently it deletes your unsaved changes - that seemed like the most obvious approach to me. I'll switch this out.